### ISSUE TITLE
Setting status date / data date

### ISSUE THREAD

**Issue by patsrik** [joniles/mpxj#13]
This class is not setting the Data date / Status date which is set in the Project Properties.

**Comment by joniles** [comment]
What type of schedule file are you reading?

**Comment by patsrik** [comment]
We are trying to generate an XML using mpxj that can be imported in Primavera. We are able to import the file but are not sure how to set the DataDate that is used in Primavera P6. The project is being set the property of status date and current date , but still the generated xml is not having those properties

```
ProjectFile file = new ProjectFile();
ProjectProperties header = file.getProjectProperties();
header.setProjectTitle("My Project");
header.setCompany("My Company");
header.setAuthor("Patsrik");
SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd");
Date startDate = format.parse("2016-06-16");
header.setStatusDate(startDate);
header.setCurrentDate(startDate);
```

**Comment by joniles** [comment]
This is how Microsoft define the status date in MS Project https://support.office.com/en-gb/article/Set-the-status-date-41359205-4860-42e6-9b7b-2d702dcb52e3 does this concept map on to how Primavera uses the data date attribute?

**Comment by patsrik** [comment]
From the link that you gave about MS Project's status date, the Data Date also behaves similarly in P6.

**Comment by joniles** [comment]
OK, I've made the change. I'm not planning to release a new version of MPXJ for a little while yet. How are you using it? Plain JAR file, Maven, NuGet?

**Comment by patsrik** [comment]
We are using the plain jar file. As of now we are using *mpxj-5.2.2.jar*

On Thu, Jun 16, 2016 at 7:40 PM, Jon Iles <notifications@github.com> wrote:

> OK, I've made the change. I'm not planning to release a new version of
> MPXJ for a little while yet. How are you using it? Plain JAR file, Maven,
> NuGet?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/joniles/mpxj/issues/13#issuecomment-226558844>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AGztUI98GvQ9qrMHtL3CKcd1tyD3XbBhks5qMYqZgaJpZM4I2Xhg>
> .
>



-- 
Regards,
Srikanth Patruni
Sr Developer, Lean Station
www.leanstation.com.sg
Mob : +49 1517 5954915

**Comment by joniles** [comment]
I've attached a jar file with the changes. Let me know how you get on, the changes will be in the next release.

[mpxj.jar.zip](https://github.com/joniles/mpxj/files/319993/mpxj.jar.zip)

**Comment by patsrik** [comment]
Really thank you for doing this. I would definitely let you know once we
test it.

On Fri, Jun 17, 2016 at 9:58 AM, Jon Iles <notifications@github.com> wrote:

> Closed #13 <https://github.com/joniles/mpxj/issues/13>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/joniles/mpxj/issues/13#event-695632286>, or mute the
> thread
> <https://github.com/notifications/unsubscribe/AGztUJ8T7h5dPB5q3ZFjlNPzvSaO62pmks5qMlOXgaJpZM4I2Xhg>
> .
>



-- 
Regards,
Srikanth Patruni
Sr Developer, Lean Station
www.leanstation.com.sg
Mob : +49 1517 5954915

**Comment by patsrik** [comment]
This new jar works perfectly fine. Thank you
